docs/src/modules/components/branding/SearchAppBar.js

```diff
@@ -0,0 +1,135 @@
+    display: 'none',
+    [theme.breakpoints.up('lg')]: {
+      display: 'block',
+    },
+    flexGrow: 1,
+    marginLeft: 50,
+    '& > *': {
+      marginRight: theme.spacing(4),
+      marginTop: 4,
+    },
+  },
+  '& .MuiSearchAppBar-logo': {
+    flexGrow: 1,
+    paddingTop: theme.spacing(0.5),
+    [theme.breakpoints.up('lg')]: {
+      flexGrow: 0,
+    },
+  },
+  '& .MuiSearcAppBar-menuButton': {
+    display: 'block',
+    [theme.breakpoints.up('lg')]: {
+      display: 'none',
+    },
+  },
+  '& [class*="MuiToolbar-root"]': {
+    [theme.breakpoints.up('lg')]: {
+      height: 80,
+    },
+  },
+}));
+
+const DrawerList = ({ toggleDrawer }) => (
+  <div
+    style={{ width: '200px' }}
+    role="presentation"
+    onClick={toggleDrawer(false)}
+    onKeyDown={toggleDrawer(false)}
+  >
+    <List>
+      <ListItem as="a">
+        <ListItemText primary={t1('Docs')} />
+      </ListItem>
+      <ListItem as="a">
+        <ListItemText primary={t1('Material-UI X')} />
+      </ListItem>
+      <ListItem as="a">
+        <ListItemText primary={t1('Pricing')} />
+      </ListItem>
+      <ListItem as="a">
+        <ListItemText primary={t1('Templates')} />
+      </ListItem>
+      <ListItem as="a">
+        <ListItemText primary={t1('About Us')} />
+      </ListItem>
+    </List>
+  </div>
+);
+
+DrawerList.propTypes = {
+  toggleDrawer: PropTypes.func,
+};
+
+export default function SearchAppBar() {
+  const [open, setOpen] = React.useState(false);
+
+  const toggleDrawer = (inOpen) => (event) => {
+    if (event.type === 'keydown' && (event.key === 'Tab' || event.key === 'Shift')) {
+      return;
+    }
+
+    setOpen(inOpen);
+  };
+
+  return (
+    <AppBarWrapper>
+      <AppBar position="static" color="inherit" elevation={0}>
+        <Toolbar>
+          <div className="MuiSearchAppBar-logo">
+            <Logo />
+          </div>
+          <div className="MuiAppBar-linksWrapper">
+            {/* TODO: Make links and update them to correct page */}
+            <Typography noWrap component="span">
+              {t1('Docs')}
+            </Typography>
+            <Typography noWrap component="span">
+              {t1('Material-UI X')}
+            </Typography>
+            <Typography noWrap component="span">
+              {t1('Pricing')}
+            </Typography>
+            <Typography noWrap component="span">
+              {t1('Templates')}
+            </Typography>
+            <Typography noWrap component="span">
+              {t1('About Us')}
+            </Typography>
+          </div>
+          <AppSearch />
+          <IconButton
+            className="MuiSearcAppBar-menuButton"
+            edge="start"
+            color="inherit"
+            aria-label={t1('menu')}
+            onClick={toggleDrawer(true)}
+          >
+            <MenuIcon />
+          </IconButton>
+        </Toolbar>
+      </AppBar>
+      <Drawer anchor={'right'} open={open} onClose={toggleDrawer(false)}>
```

Comment:
Should we care about the difference? Historically, we have been using `"` anything we could, e.g. class names. But, I have also seen Damien use this approach, so I wonder

```suggestion
      <Drawer anchor="right" open={open} onClose={toggleDrawer(false)}>
```

Reply:
Not sure, I'd except prettier to fix it if we care about it :upside_down_face: I can replace those, but I can't guarantee I won't add them again 😄